packages/react/src/ReactFetch.js

```diff
@@ -51,12 +51,8 @@ if (enableCache && enableFetchInstrumentation) {
         // We're outside a cached scope.
         return originalFetch(resource, options);
       }
-      if (
-        options &&
-        options.signal &&
-        options.signal !== dispatcher.getCacheSignal()
```

Comment:
This suggests that maybe someone (Next?) is passing the signal returned from getCacheSignal() back to React. But since getCacheSignal isn't exposed, we should be able to just not do that?

Reply:
confirmed offline that Next is not using this API, so it's safe to remove this